otherlibs/cram/bin/dune

```diff
@@ -0,0 +1,11 @@
+(executable
+ (name dune_cram)
+ (libraries dune-private-libs.stdune
+            dune-build-info
+            cmdliner
+            build_path_prefix_map
+            dune-private-libs.dune_re
```

Comment:
Perhaps the publicly released version of cram should use cmdliner and re from opam. That would teach more about some of the bundling related problem we're trying to solve :)

Reply:
Haha maybe. But that's a problem for later :)